### DISCUSSION TITLE
Programming Instructions - Step 4

### DISCUSSION THREAD

**Discussion by chadoulis** [STELLA-Landsat/STELLA#22] [Build Notes: How do I make a STELLA?]
In step 4 of the programming instructions for Stella 2.0, one can see the following: 

`Download the .ZIP file. link to file .replace “In the repository” with “In the software download”… Remove “It’s the only file in that folder.`

It is not very clear to me what is meant. Could someone please clarify? Thanks.

**Comment by paulmirel** [comment]
That was an editing note. The instruction should read: 

Steps 4, 5, 6: 
Download the STELLA 2.0 Software ZIP. 
Unzip the file. 
Open the S2 code and libraries folder. 
Drag both the code.py file and the lib folder to drop them on the CIRCUITPY drive.
